store/copr/coprocessor.go

```diff
@@ -1360,3 +1366,21 @@ func isolationLevelToPB(level kv.IsoLevel) kvrpcpb.IsolationLevel {
 		return kvrpcpb.IsolationLevel_SI
 	}
 }
+
+func checkKeyRangeSorted(keyRanges []kv.KeyRange) bool {
```

Comment:
We can use this [function](https://pkg.go.dev/golang.org/x/exp/slices#IsSortedFunc) to simple code.

Reply:
Done